READMEs/datatokens-flow.md

```diff
@@ -18,40 +18,53 @@ Ocean `barge` runs ganache (local blockchain), Provider (data service), and Aqua
 In a new console:
 
 ```console
-#grab repo
+# Grab repo
 git clone https://github.com/oceanprotocol/barge
 cd barge
 
-#clean up old containers (to be sure)
+# Clean up old containers (to be sure)
 docker system prune -a --volumes
 
-#run barge: start ganache, Provider, Aquarius; deploy contracts; update ~/.ocean
-./start_ocean.sh  --with-provider2
+# Run barge: start ganache, Provider, Aquarius; deploy contracts; update ~/.ocean
+./start_ocean.sh
 ```
 
 ## Install the library, set envvars
 
 In a new console:
 
 ```console
-#Initialize virtual environment and activate it.
-python -m venv venv
+# Grab ocean.py repo
+cd Desktop/
+git clone https://github.com/oceanprotocol/ocean.py.git
+git checkout v4main
+
+# Create your working directory. Copy artifacts.
+mkdir test3
+cd test3
+cp -r ../ocean.py/artifacts ./
```

Comment:
this will not work, since the installation doesn't pull over artifacts, yet. Only the ocean_lib part. That's why we also don't use any imports from tests. We need to adapt the installation part and that is an ongoing discussion. Still it is not an impediment to merge this. We will probably update all the installation parts later. @mariacarmina can you please create a ticket for this? I think we don't have one yet.

Reply:
It works if you clone the repo ocean.py and checkout with v4 (https://github.com/oceanprotocol/ocean.py/tree/v4main/artifacts). You copy the artifacts folder from ocean.py folder and paste into test3. Intermediary it works.